src/core/radio/radio.cpp

```diff
@@ -92,4 +95,130 @@ Error Radio::Transmit(Mac::TxFrame &aFrame)
 }
 #endif // OPENTHREAD_CONFIG_RADIO_LINK_IEEE_802_15_4_ENABLE
 
+#if OPENTHREAD_CONFIG_RADIO_TIME_STATS_ENABLE && (OPENTHREAD_FTD || OPENTHREAD_MTD)
+inline uint64_t UintSafeMinus(uint64_t aLhs, uint64_t aRhs)
+{
+    uint64_t result = 0;
+    if (aLhs > aRhs)
+    {
+        result = aLhs - aRhs;
+    }
+    return result;
```

Comment:
Simplified to `return (aLhs > aRhs) ? (aLhs - aRhs): 0;` ?

Reply:
+1